src/main/java/frc/robot/subsystems/Camera/Camera.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.camera;
+
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonPipelineResult;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+public class Camera extends SubsystemBase {
+  PhotonCamera camera;
+
+  public Camera() {
+    this.camera = new PhotonCamera("slotheye");
+  }
+
+  @Override
+  public void periodic() {
+    
+  }
+
+  @Override
+  public void simulationPeriodic() {
+    // This method will be called once per scheduler run during simulation
+  }
+
+  public PhotonPipelineResult getCameraResult() {
+    PhotonPipelineResult result = camera.getLatestResult();
+    return result;
+  }
+
+  public PhotonTrackedTarget getCameraTarget() {
+    PhotonPipelineResult result = camera.getLatestResult();
```

Comment:
Since you have a `getCameraResult()` method, you can use that to get subsequent results. That way if it needs changes you only need to change it in one place.